File: FizzBuzz.fs
open System
let fizzBuzz n =
    [1..n]
    |> List.iter (fun i ->
        match i % 3, i % 5 with
        | 0, 0 -> printfn "FizzBuzz"
        | 0, _ -> printfn "Fizz"
        | _, 0 -> printfn "Buzz"
        | _ -> printfn "%d" i)

[<EntryPoint>]
let main argv =
    printfn "Enter the value of n:"
    let n = Console.ReadLine() |> int

    // Call the fizzBuzz function with the given n
    fizzBuzz n

    0 // return an integer exit code

File: FizzBuzz.go
package main

import "fmt"

func fizzBuzz(n int) {
	for i := 1; i <= n; i++ {
		switch {
		case i%3 == 0 && i%5 == 0:
			fmt.Print("FizzBuzz ")
		case i%3 == 0:
			fmt.Print("Fizz ")
		case i%5 == 0:
			fmt.Print("Buzz ")
		default:
			fmt.Printf("%d ", i)
		}
	}
}

func main() {
	var n int
	fmt.Print("Enter the value of n: ")
	fmt.Scan(&n)

	// Call the fizzBuzz function with the given n
	fizzBuzz(n)
}
